doc/pages/highlighters.asciidoc

```diff
@@ -218,15 +218,15 @@ add-highlighter <path>/<name>/<region_name> region \
 *recurse*::
     regex that defines the text that matches recursively an end token
     into the region, every match of *recurse* will consume a following
-    match of *end*, preventing it from closing the region.
+    match of *closing* regex, preventing it from closing the region.
 
```

Comment:
This is really the main edit in this PR as there is no parameter called `*end*`. We should write `*closing*` to be consistent with the parameter listing above...